scripts/seed-database/write/users.sql

```diff
@@ -7,6 +7,8 @@ CREATE TEMPORARY TABLE sync_users (
   created_at timestamptz,
   updated_at timestamptz,
   is_platform_admin boolean
+  -- ,
+  -- is_staging_only boolean
```

Comment:
are these lines intended to be comment-out in sync script?

Reply:
Yep - it's a bit of chicken and egg situation - we can't sync from the new column until it exists on prod.

Once this is across there'll be a quick PR uncommenting these lines (plus any potential sync script fixes as it's currently untestable).

Reply:
(And the catch here is that pizzas will fail unless it's commented out as they can't sync)